.github/workflows/hackathon.yml

```diff
@@ -85,3 +85,38 @@ jobs:
           HELICONE_CACHE_ENABLED: false
           HELICONE_PROPERTY_AGENT: ${{ matrix.agent-name }}
           REPORT_LOCATION: ${{ format('../../reports/{0}', matrix.agent-name) }}
+
+      # Support node.js-based agents.
+      # NOTE: This is for example purposes, you will need to move this logic wherever you see fit
+
+      - name: Read node version from .nvmrc (or use default v18.15)
+        id: nvm
+        run: echo NVMRC=$([[ -f .nvmrc ]] && cat .nvmrc || echo "v18.15") >> "$GITHUB_OUTPUT"
+        working-directory: ${{ matrix.agent-name }}
```

Comment:
I'm assuming here that each agent will be cloned at the root directory, and put under a `./${agent-name}` directory